src/introduction/basic-concepts.md

```diff
@@ -179,6 +179,7 @@ vendors.
 
 - Semantic Versioning - <https://semver.org/>
 - The CWL Specification page in the CWL website: <https://www.commonwl.org/specification/>
+    - [See the Command Line Tool Description Standard](https://www.commonwl.org/v1.2/CommandLineTool.html)
```

Comment:
If we want to include this - we probably want to format it like the other entries in this list, so it would be description and then link.

Reply:
thank YOU @swzCuroverse